examples/basic/README.md

```diff
@@ -0,0 +1,23 @@
+# Basic Backend Example
+Creates resources for a secure backend in AWS
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+No requirements.
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| aws | n/a |
```

Comment:
Need a version specified

Reply:
I'm not wasting time perfecting examples

Reply:
🙄